homeassistant/components/blebox/binary_sensor.py

```diff
@@ -0,0 +1,62 @@
+"""BleBox sensor entities."""
+from dataclasses import dataclass
+
+from blebox_uniapi.binary_sensor import BinarySensor as BinarySensorFeature
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import BleBoxEntity, create_blebox_entities
+
+
+@dataclass
+class BleboxBinarySensorEntityDescription(BinarySensorEntityDescription):
+    """Class describing Blebox binary sensor entities."""
+
+
```

Comment:
This class doesn't add anything that `BinarySensorEntityDescription` doesn't already provide.
```suggestion
```